### DISCUSSION TITLE
Next.js SSR example?

### DISCUSSION THREAD

**Discussion by djMax** [LegendApp/legend-state#64] [Q&A] [answered]
I think I have a sense of how this is likely to work, but could you provide some guidance on how to handle SSR rehydration on the client? Is it as simple as just making sure observable() is called on both sides and that the server sends down the POJO of the observed value somewhere (getInitialProps)?

**Comment by jmeistrich** [comment, marked as answer]
Yes, it should not need to work any differently than any other kind of state. As long as the value in the observable matches on client and server, everything should work as expected.
